docs/concepts/flows.md

```diff
@@ -1,44 +1,67 @@
 ---
 order: 1
 ---
+
 # Flow
-A [flow](../developer-guide/flow) is a simple list of tasks that describe the behavior of your flow. Flows are grouped by namespaces.
 
-## Task
-A Task is a single action in a flow. 
-A task can optionally take inputs, perform an action, or produce an output.
+A [flow](../developer-guide/flow) is a list of tasks. You create flows in Kestra to automate your processes.
```

Comment:
Here you remove the mention to namespace, in this case the Namesapce section would be better just under.
or you can add something like "Flows are grouped in namespaces"

Reply:
The rationale excludes terms that have been introduced or aren't defined immediately after. Also, the definition of flows is independent of namespaces, hence the omission.